core/embed/util/image/image.c

```diff
@@ -264,7 +264,6 @@ secbool check_image_contents(const image_header *const hdr, uint32_t firstskip,
 
   // Check the firmware integrity, calculate and compare hashes
   size_t offset = IMAGE_CODE_ALIGN(firstskip);
```

Comment:
this should probably be `const size_t something_else = ...` ...

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/c4fe8423870981420e7da280af9b1d5f871fbfe8